lambdatimes/src/components/Content/Content.js

```diff
@@ -17,29 +17,44 @@ export default class Content extends Component {
   }
 
   componentDidMount() {
-    // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState({
+      tabs: tabData,
+      cards: cardData
+    });
   }
 
   changeSelected = tab => {
-    // this function should take in the tab and update the state with the new tab.
+    this.setState({
+      selected: tab
+    })
   };
 
   filterCards = () => {
     /* Right now this function only returns the cards on state.
       We're going to make this function more dynamic
       by using it to filter out our cards for when a tab is selcted
       
-      Notice that we're passing this function to our <Cards /> component below.
+      Notice that we're passing what this function returns to our <Cards /> component below.
       This function returns an array of cards, so we can just pass it down as such.
 
       Your algorithim for the logic here is as follows: 
         - if the selected tab is 'all' it should return all 
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
-    return this.state.cards;
+    let cardSet = this.state.cards.filter(card => {
```

Comment:
Could do this as a one liner, because filter only needs to return truthy values
let cardSet = this.state.cards.filter(card => (this.state.selected === 'all' || this.state.selected === card.tab));